internal/pkg/agent/install/uninstall_windows_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build windows
+
+package install
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const simpleBlockForever = `
+package main
+func main() {
+    select{}
+}
+`
```

Comment:
When I try to run this program (admittedly on my Mac), it exits pretty much immediately citing a deadlock:

```
$ time go run block.go
fatal error: all goroutines are asleep - deadlock!

goroutine 1 [select (no cases)]:
main.main()
	/tmp/block.go:4 +0x1c
exit status 2
go run block.go  0.05s user 0.06s system 59% cpu 0.193 total
```

Is it possible the same thing could happen in your test before `RemovePath` gets called?

Reply:
I ran the tests with the debugger to validate that I was getting the error handling correct and that the code actually works, using the same code.

But I do see that same error, so I have update the test to sleep forever.